sgf-viewer/public/sgfs/amateur_vs_advh_498mil/tony-vs-dragonslayer-497mil.sgfs

```diff
@@ -0,0 +1 @@
+(;GM[1]FF[4]CA[UTF-8]AP[Sabaki:0.52.0]KM[6.5]SZ[19]DT[2022-11-15]RE[W+43.5]PB[Adversary]PW[Victim];B[qg];W[dp];B[pj];W[dd];B[ee];W[de];B[cc];W[cd];B[ef];W[dg];B[pd];W[pp];B[eg];W[eh];B[fh];W[ei];B[fg];W[fi];B[hh];W[hi];B[qm];W[on];B[qn];W[po];B[gh];W[gi];B[rp];W[qq];B[rq];W[ml];B[qr];W[pr];B[rr];W[ii];B[nc];W[ki];B[bi];W[ci];B[bj];W[bk];B[bf];W[bh];B[bg];W[ch];B[ah];W[cf];B[cg];W[df];B[cj];W[dj];B[di];W[dh];B[ej];W[dk];B[ck];W[cl];B[dl];W[bl];B[ek];W[di];B[ai];W[be];B[bd];W[bc];B[el];W[cm];B[bb];W[ad];B[ag];W[fm];B[cb];W[em];B[ac];W[bd];B[hj];W[ij];B[fj];W[gl];B[ik];W[jk];B[kj];W[jj];B[gj];W[il];B[jl];W[hk];B[kk];W[kl];B[hl];W[hm];B[li];W[ji];B[dc];W[ed];B[ec];W[fd];B[fc];W[gd];B[gc];W[ge];B[ic];W[ig];B[gf];W[hf];B[ih];W[jh];B[hg];W[if];B[kh];W[jg];B[km];W[jm];B[jf];W[kg];B[kd];W[lh];B[lg];W[lf];B[me];W[lj];B[mg];W[mi];B[kf];W[lk];B[nh];W[ni];B[oh];W[oj];B[pk];W[ok];B[oi];W[nj];B[qc];W[pl];B[ql];W[pm];B[fe];W[he];B[id];W[je];B[ke];W[ie];B[jd];W[ak];B[or];W[nr];B[ps];W[pq];B[mr];W[oq];B[os];W[nq];B[ns];W[kr];B[mq];W[mp];B[kq];W[jr];B[lq];W[jp];B[jq];W[iq];B[ms];W[ir];B[ab];W[kp];B[lp];W[lo];B[lr];W[ks];B[qo];W[qp];B[qs];W[ls];B[pn];W[np];B[mh];W[hd];B[hc];W[kb];B[jb];W[ka];B[ja];W[lb];B[lc];W[mb];B[nb];W[mf];B[mc];W[pi];B[qi];W[ph];B[pg];W[ng];B[qh];W[nf];B[ne];W[og];B[kc];W[pi];B[ph];W[kn];B[ma];W[cq];B[of];W[rb];B[le];W[qb];B[rc];W[pb];B[sb];W[sd];B[sc];W[ra];B[pc];W[gp];B[ob];W[dm];B[];W[])
```

Comment:
I've been calling it 497mil in all my experiments, but since the step number is 497.7mil, I rounded it up to 498mil in the website text. I'll rename the directory to 497mil for consistency with the experiments I guess